{"name": "test_user_login_with_incorrect_data", "status": "passed", "description": "Логин с неверным логином и паролем", "start": 1724228128735, "stop": 1724228129765, "uuid": "70f0db5a-51c6-4706-b24c-dc02c1a0fc8e", "historyId": "88e9223a586f217712e0f1194e40759e", "testCaseId": "88e9223a586f217712e0f1194e40759e", "fullName": "tests.test_user_login#test_user_login_with_incorrect_data", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user_login"}, {"name": "host", "value": "DESKTOP-0A2OPDH"}, {"name": "thread", "value": "18500-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user_login"}]}